### PR TITLE
Richard severich spin words

### PR DESCRIPTION
**Pull request by RichardSeverich**
This is my firs attempt to solve the problem SpinWord.

### CODE REVIEW

File: src/test/java/org/fundacionjala/coding/richard/severich/spinwords/SpinWordsTest.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        String expectedResult = "This is a test";
+        assertEquals(expectedResult, spinWords.spin("This is a test"));
+    }
+
+    /**
+     * This test checks the string that returns the method getSpinWord returns the expected string.
+     */
+    @Test
+    public void verifyChangeThisIsAnotherTest() {
+        SpinWords spinWords = new SpinWords();
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/richard/severich/spinwords/SpinWordsTest.java]: You are repeating this code in all your tests
Please put them in @before method or convert it to a static method.

File: src/main/java/org/fundacionjala/coding/richard/severich/spinwords/SpinWords.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    /**
+     * This is the method that overturns strings that are more than 4 characters long.
+     *
+     * @param spinWord It is the initial string.
+     * @return Is the final string.
+     */
+    public String spin(String spinWord) {
+
+
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/coding/richard/severich/spinwords/SpinWords.java]: Remove unnecessary blank lines

**Comment by RichardSeverich** [comment]
Richard_Severich_Spin_Words with corrections.

**Comment by carledriss** [comment]
LGTM!

**Pull request merged**